Inferences/FastCondensation.cpp

```diff
@@ -71,7 +71,7 @@ Clause* FastCondensation::simplify(Clause* cl)
 {
   CALL("FastCondensation::perform");
 
-  TimeCounter tc(TC_CONDENSATION);
+  TIME_TRACE("condensation");
```

Comment:
Does this want to be the same `"condensation"` as slow-condensation? If so should be `static const char *`, if not maybe could be `"fast condensation"`?

Reply:
yeah true .. i wasn't sure about that so i did the only wrong thing apparently hehe
I'll change it to `"fast condensation"`